### PR TITLE
Travis CI: Add pytest --doctest-modules file_transfer_protocol

### LINKED ISSUE #983: pytest --doctest-modules file_transfer_protocol fails
Missing __if \_\_name\_\_ == "\_\_main\_\_":__ ?

@AlexDvorak, @BoboTiG, @louisparis would you be willing to fix this?

### PR DESCRIPTION
**Pull request by cclauss**
Fixes #983  The two Python files in the __file_transfer_protocol__ directory currently fail the Travis CI pytests.  This pull request makes the changes necessary for Travis CI to run pytest on these files without hanging.

### CODE REVIEW

File: file_transfer_protocol/ftp_client_server.py [hunk 1]
```diff
@@ -2,56 +2,55 @@
...
+                conn.send(in_data)
+                print('Sent ', repr(in_data))
+                in_data = f.read(1024)
+
+        print('Done sending')
+        conn.send('Thank you for connecting')
+        conn.close()
+
+
+    # client side server
```
[the remark this answers is not in the document]
Comment (AlexDvorak) [reply file_transfer_protocol/ftp_client_server.py]: given the loop above, I think this should be in a separate file

**Comment by cclauss** [comment]
@AlexDvorak I agree.  This PR was just trying to get the pytests to run across the entire repo.

Please create a _separate_ pull request that deals with all of the issues that you have raised.  I will gladly review that PR when it passes our Travis CI tests and then close this PR in favor of yours

**Comment by cclauss** [comment]
@AlexDvorak Your thoughts on progressing this in a separate PR?

**Comment by AlexDvorak** [comment]
That sounds good, I should have the PR open by tomorrow.
  

  
  

  
  
>

**Pull request closed without merging**